jsapp/scss/stylesheets/partials/_registration.scss

```diff
@@ -464,6 +464,13 @@
       > .password2 {
         flex: 0; // Let these be shorter to match designs
       }
+      > .newsletter_subscription {
+        input[type='checkbox'] {
+          float: left;
+          margin-right: 10px;
+          margin-top: -8.5px;
+        }
+      }
```

Comment:
Cool! There are some refinements we could make here for checkbox alignment, but this is good for now. The newsletter and ToS checkboxes will eventually go below the password confirmation, and at that point we can nudge the checkboxes to match the designs a little more.